screenshots/build.js

```diff
@@ -0,0 +1,17 @@
+var page = require('webpage').create();
+page.viewportSize = {width: 1000, height: 1000};
+page.open('./test/perceptual/alert.html', function() {
+	var clipRect = page.evaluate(function () {
+		var screenshot = document.querySelector(".screenshot");
+		screenshot.style.display = 'inline-block';
+		return screenshot.getBoundingClientRect();
+	});
+	page.clipRect = {
+		top:    clipRect.top,
+		left:   clipRect.left,
+		width:  clipRect.width,
+		height: clipRect.height
+	};
+	page.render('screenshot.png');
```

Comment:
I think we want the actual screenshots in the `screenshots` folder.  For repos with single components it's clean enough, but for repos that may have several components I personally find the root of the repo starts getting messy.  Others may have different opinion.